layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi_6.6.bb

```diff
@@ -0,0 +1,33 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/../../../meta-raspberrypi/recipes-kernel/linux/files:"
```

Comment:
Is there a better way to do this?

Reply:
@alexgg I've considered that, but copying the files here would create an override for all the device types, not just the Pi5 and I wanted to keep the changes as self-contained as possible. We'll get rid of this recipe anyway with the update to scarthgap, ideally we should just `bbappend` the version and commit hashes.

Reply:
copy them into a `files/raspberrypi5` directory @mtoman and they will only get used by the raspberrypi5

Reply:
Done